src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldLikeEventSymbol.cs

```diff
@@ -114,8 +114,9 @@ internal SourceFieldLikeEventSymbol(SourceMemberContainerTypeSymbol containingTy
             }
 
             // Accessors will assume that Type is available.
-            _addMethod = new SynthesizedEventAccessorSymbol(this, isAdder: true);
-            _removeMethod = new SynthesizedEventAccessorSymbol(this, isAdder: false);
+            // No body, as this was declared by a event-field, nothing that has an actual block or expression body.
+            _addMethod = new SynthesizedEventAccessorSymbol(this, isAdder: true, hasAnyBody: false, isExpressionBodied: false);
```

Comment:
>false [](http://example.com/codeflow?start=93&length=5)

I think this still doesn't match what was recommended at [https://github.com/dotnet/roslyn/pull/68158#discussion_r1191649763](https://github.com/dotnet/roslyn/pull/68158#discussion_r1191649763).

Specifically: "it looks like SynthesizedEventAccessorSymbol **should have a body unless it is abstract**." Is this event always abstract? These accessors are synthesized, there is no syntax for them, but, if they are not abstract, they have body.

Reply:
I'm not sure what body these have... they are fields, not something with a body afaict.  Is the flag only a representation on whether something is abstract or not?  If so, would it make more sense to call it isAbstract (or isNotAbstract)?  Right now it seems hard to tell what actual semantics this wants.

Reply:
> I'm not sure what body these have :). 

That is Ok. That is why I am suggesting you what value to use. You are not following the suggestion, but responding to it with: "Done." 
```SynthesizedEventAccessorSymbol``` represents a synthesized accessor, it has a body, unless it is abstract, Not expression body because that is syntax specific concept and we don't have syntax for the accessor. This fallows the approach we used and kept using with other synthesized methods. I am suggesting to pass ```!IsAbstract``` for ```hasAnyBody```. We don't want to rename anything, or change  meaning of anything at the moment.

Reply:
> they are fields

The ```SourceFieldLikeEventSymbol``` represents an event. An event can be abstract.

Here is an abstract field-like event:
```
abstract class C
{
    public abstract event System.Action E;
}
```

Reply:
>  I am suggesting to pass !IsAbstract for hasAnyBody. We don't want to rename anything, or change meaning of anything at the moment.

My point is: this would be much clearer and easier to implement (and maintain) if the naming indicates what the concept represents.

Reply:
I already expressed my position.

Reply:
moved and doc'ed.